providers/nutshell/handlers.go

```diff
@@ -0,0 +1,144 @@
+package nutshell
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/common/urlbuilder"
+	"github.com/amp-labs/connectors/internal/datautils"
+	"github.com/amp-labs/connectors/internal/jsonquery"
+	"github.com/amp-labs/connectors/providers/nutshell/internal/metadata"
+	"github.com/spyzhov/ajson"
+)
+
+// The API accepts large page numbers without an error.
+// This value (5,000) was chosen arbitrarily.
+// The actual maximum limit is 10,000; anything higher will return an error.
+const defaultPageSize = "5000"
```

Comment:
if there are no performance implications, can we set this to the max? (9999 or 10000)